hack/tools/tools.go

```diff
@@ -28,3 +28,5 @@ import (
 	// gen-crd-api-reference-docs
 	_ "github.com/ahmetb/gen-crd-api-reference-docs"
 )
+
+
```

Comment:
```suggestion

```

https://github.com/kubernetes-sigs/kwok/blob/6d51ec0f44152a4b13e532a416ecdaca45434ef5/hack/verify-go-format.sh#L24

It looks like format's check didn't cover this file 😓